Enterprise-CMCS/macpro-mdct-mcr
# User Zustandification™️

### Description

Converting the `UserContext` state management of MCR from `Context API` to `Zustand` (following the patterns implemented in MFP).

### Related ticket(s)


CMDCT-2794

---
### How to test

Running this locally, you should be able to log in, log out, and manage Banners (as an Admin) as usual.

Optionally: install the [Redux Devtools Chrome extension](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd) or the [Firefox Redux Devtools extension](https://addons.mozilla.org/en-US/firefox/addon/reduxdevtools/)

- Open devtools
- Log in
- You should see the `user` object on the extension ✨

<img width="407" alt="Screenshot 2023-11-27 at 2 31 09 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/99458559/f33d79a4-5509-45a0-b08e-4e0d1a338fe2">

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
